s2d-oms-mgmt-solution/azuredeploy.json

```diff
@@ -0,0 +1,1847 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "omsLogAnalyticsWorkspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of existing OMS Log Analytics workspace."
+      }
+    },
+    "omsLogAnalyticsWorkspaceLocation": {
+      "type": "string",
+      "allowedValues": [
```

Comment:
@gatneil Of all my PRs only 1-2 show the error about the additional parameter. This was cause because I've copied some files at my last tries to test something. All other PRs does not show any information what is the error about. The error is this:
```json
Server Error:{
    "error": "error: Deployment provisioning state was not successful\nerror: The resource operation completed with terminal provisioning state 'Failed'.\n",
    "_rgName": "qstci-2c212186-17b8-8a49-67e4-22038bdf1d2d",
    "command": "azure group deployment create --resource-group (your_group_name) --template-file azuredeploy.json --parameters-file azuredeploy.parameters.json",
    "parameters": {
        "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
        "contentVersion": "1.0.0.0",
        "parameters": {
            "omsLogAnalyticsWorkspaceName": {
                "value": "ci1dfa41c39c0993f5"
            },
            "omsLogAnalyticsWorkspaceLocation": {
                "value": "West Europe"
            }
        }
    },
```
This is the same error for the latest PRs I've done today. Basically no information what is causing the error. To me there are 3 reasons for that:
- Build system cannot get the actual error.
- The api is not giving proper error
- The build system does not deploys properly as in my tests it deploys more than fine (see screenshot)

![image](https://cloud.githubusercontent.com/assets/11890856/21603255/66505dc4-d1a4-11e6-8b1d-a08fc47f4fb6.png)